examples/data/losses/analysis_losses.ini

```diff
@@ -0,0 +1,17 @@
+[project]
+name = republic_dominican
+
+[analysis1]
+name = republic_dominican_redundancy
+analysis = single_link_losses
+weighing = time
+duration_event = 600
+production_loss_per_capita_per_day = 1000
+part_of_day = day
+hours_per_day = 10
+trip_purposes = business,commute,freight,other
+traffic_intensities_file = C:\Users\asgarpou\OneDrive - Stichting Deltares\Documents\Projects\Long involvement\Ra2ce development\2023\sprint_november\Losses module\2_input_data\traffic_intensities.csv
```

Comment:
These paths should not point to your C drive here, otherwise the notebook will crash in the binder.
Maybe set them to None in the .ini and deroute the paths manually in the Jupyter notebook